src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/EnumConverter.cs

```diff
@@ -20,12 +21,16 @@ public class EnumConverter : TypeConverter
         /// Initializes a new instance of the <see cref='System.ComponentModel.EnumConverter'/> class for the given
         /// type.
         /// </summary>
-        public EnumConverter([DynamicallyAccessedMembers(TypeDescriptor.ReflectTypesDynamicallyAccessedMembers)] Type type)
+        public EnumConverter(Type type)
         {
+            if (!type.IsEnum && !type.Equals(typeof(Enum)))
+            {
+                throw new ArgumentException(SR.EnumInvalidValue);
```

Comment:
Is this an error case you found?

Reply:
When would someone pass in `typeof(Enum)`?

Reply:
Yep, the abstract Enum class seems to be supported in `EnumConverter` and there are [tests ](https://github.com/dotnet/runtime/blob/main/src/libraries/System.ComponentModel.TypeConverter/tests/EnumConverterTest.cs#L35)for this.